sup3r/preprocessing/conditional_moment_batch_handling.py

```diff
@@ -128,6 +134,12 @@ def make_mask(high_res, s_padding=None, t_padding=None):
         t_padding : int | None
             Temporal padding size. If None or 0, no padding is applied.
             None by default
+        end_t_padding : bool | False
```

Comment:
Could you expand this description a little. I don't totally understand the purpose of this arg.